src/Diagnostics.RuntimeHost/Utilities/DateTimeHelper.cs

```diff
@@ -103,9 +104,14 @@ internal static bool PrepareStartEndTimeUtc(string startTime, string endTime, ou
                 errorMessage = "Invalid Start Time and End Time. End Time cannot be earlier than Start Time.";
                 return false;
             }
-            else if (startTimeUtc > currentUtcTime)
+            if (endTimeUtc > currentUtcTime.AddMinutes(kustoDelayInMinutes))
             {
-                errorMessage = "Invalid Start Time. Start Time cannot be a future date.";
+                errorMessage = $"Invalid Start Time. End Time should be at least {HostConstants.KustoDelayInMinutes} minutes from now";
```

Comment:
Possibly change the error message to 
"Invalid Start Time. End Time should be less by at least {HostConstants.KustoDelayInMinutes} minutes from now"